src/main/java/org/breedinginsight/utilities/email/EmailUtil.java

```diff
@@ -40,13 +38,29 @@ public class EmailUtil {
     private Integer smtpHostPort;
     @Property(name = "email.from")
     private String fromEmail;
+    @Property(name = "email.relay-server.login")
+    private String smtpLogin;
+    @Property(name = "email.relay-server.password")
+    private String smtpPassword;
 
     private Session getSmtpHost() {
         Properties props = new Properties();
         props.put("mail.smtp.host", smtpHostServer);
         props.put("mail.smtp.port", smtpHostPort);
         props.put("mail.debug", true);
-        return Session.getInstance(props, null);
+        Authenticator auth = null;
+        if (Utilities.isNeitherNullNorEmpty(smtpLogin) && Utilities.isNeitherNullNorEmpty(smtpPassword)) {
```

Comment:
I think you could use `StringUtils.isNotBlank` from apache commons if you wanted to use an existing implementation.

Reply:
Done.